### ISSUE TITLE
[Improvement] Quote style

### ISSUE THREAD

**Issue by anenadic** [carpentries/workbench#26]
Quotes are not that visually different from other text as shown in the image below:

<img width="744" alt="image" src="https://user-images.githubusercontent.com/631719/181015748-684fdae4-be32-4a44-81f1-a932c90cbecc.png">

The font is bigger but it is not necessarily clear this is a quote. I know this is matter of personal preference.

**Comment by zkamvar** [comment]
Can you give me a link to the page where you are seeing this? The blockquote issue was reported in https://github.com/carpentries/varnish/issues/27 and fixed in [varnish 0.1.11](https://carpentries.github.io/varnish/news/index.html#varnish-0111):

![screencapture of a blockquote that is indented and has a grey vertical bar offsetting it from the rest of the text](https://user-images.githubusercontent.com/3639446/183139761-67046460-33e6-4041-a15e-1d1046ff2bb2.png)

If this is local, you should try updating varnish:

```r
install.packages("sandpaper", repos = c("https://carpentries.r-universe.dev/", getOption("repos")))
sanpaper::update_varnish()
```

**Comment by zkamvar** [comment]
Hi @anenadic, do you still see this issue?
